inventory.py

```diff
@@ -8,9 +8,10 @@
 Generates Ansible inventory with hostnames from a static inventory file located in the same dir as this script.
 By default this script looks for an inventory named 
     inventory.ini
- (default) or alternatively from a file defined in
-    export AI_INVENTORY='some_inventory.ini'
-Optionally the hostnames can be prefixed with one of our proxy/jumphost servers.
+or alternatively for an inventory file name as defined in
+    export AI_INVENTORY='some_inventory.ini
+
+The hostnames parsed from the static inventory file can be prefixed with the hostname of one of our proxy/jumphost servers.
```

Comment:
Denk even om de regellengte horizontaal scrollen is niet prettig in python files.